src/basic/strxcpyx.c

```diff
@@ -57,7 +57,7 @@ size_t strpcpyf(char **dest, size_t size, const char *src, ...) {
                 *dest += i;
                 size -= i;
         } else {
-                *dest += size;
+                *dest += size - 1;
```

Comment:
So this function is generally stupid, because `vsnprintf` null-terminates the buffer. So, apart from the error you fixed, line 64 should be removed too.